reana_job_controller/kubernetes_job_manager.py

```diff
@@ -56,6 +56,8 @@ def __init__(self, docker_img=None, cmd=None, env_vars=None, job_id=None,
         :type shared_file_system: bool
         :param job_name: Name of the job.
         :type job_name: str
+        :param kerberos: Provide Kerberos for job container.
```

Comment:
This description reads better https://github.com/reanahub/reana-commons/pull/135/files#diff-859c5527dd4c2157756a43e8d3355b31R85.